posthog/api/test/dashboards/test_dashboard.py

```diff
@@ -21,6 +21,13 @@
 class TestDashboard(APIBaseTest, QueryMatchingTest):
     def setUp(self) -> None:
         super().setUp()
+
+        self.organization.available_features = [
+            AvailableFeature.DASHBOARD_COLLABORATION,
+            AvailableFeature.DASHBOARD_PERMISSIONING,
+        ]
```

Comment:
Considering `test_cannot_duplicate_dashboard_with_no_permissions_to_edit_it` requires these, it should probably be in `ee/api/test/test_dashboard.py` (there all the features are already available)